tests/unit/template_cassdc_test.go

```diff
@@ -171,6 +173,66 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 			Expect(err).To(HaveOccurred())
 		})
 
+		It("enable Cassandra auth", func() {
+			dcName := "test"
+			clusterSize := 3
+			authCachePeriod := int64(7200000)
+			options := &helm.Options{
+				KubectlOptions: defaultKubeCtlOptions,
+				SetValues: map[string]string{
+					"k8ssandra.datacenterName":                                      dcName,
+					"k8ssandra.size":                                                strconv.Itoa(clusterSize),
+					"k8ssandra.configuration.auth.enabled":                          "true",
+					"k8ssandra.configuration.auth.caches.rolesValidityMillis":       strconv.FormatInt(authCachePeriod, 10),
+					"k8ssandra.configuration.auth.caches.rolesUpdateMillis":         strconv.FormatInt(authCachePeriod, 10),
```

Comment:
Should these be `+1`, `+2` etc to verify that we actually set the correct value in the helm chart? At the moment this wouldn't catch it if there's a copy-paste error from another line.